samples/meeting-tabs/python/components/app-in-meeting.html

```diff
@@ -0,0 +1,318 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>App in Meeting</title>
+    <script src="https://unpkg.com/@microsoft/teams-js@2.0.0/dist/MicrosoftTeams.min.js"></script> <!-- Teams SDK -->
```

Comment:
Update Team-js SDK CDN Link to latest and test the sample.

Latest version
**@microsoft/teams-js@2.40.0**